Sources/ShopifyCheckoutSheetKit/PixelEvents.swift

```diff
@@ -972,33 +853,12 @@ extension PixelEventsPaymentInfoSubmittedData {
 	}
 }
 
-// MARK: - CustomEvent
-
-/// This event represents any custom events emitted by partners or merchants via
-/// the `publish` method
-public struct CustomEvent: Codable {
-	/// The client-side ID of the customer, provided by Shopify
-	let context: Context?
-	let customData: CustomData?
-	/// The ID of the customer event
-	let id: String?
-	/// Arbitrary name of the custom event
-	let name: String?
-	/// The timestamp of when the customer event occurred, in [ISO
-	/// 8601](https://en.wikipedia.org/wiki/ISO_8601) format
-	let timestamp: String?
-
-	enum CodingKeys: String, CodingKey {
-		case context, customData, id, name, timestamp
-	}
-}
-
 // MARK: CustomEvent convenience initializers and mutators
 
 extension CustomEvent {
 	init(from webPixelsEventBody: WebPixelsEventBody) {
 		self.context = webPixelsEventBody.context
-		self.customData = webPixelsEventBody.customData
+		self.data = webPixelsEventBody.customData
```

Comment:
this might be unexpected by developers who are familiar with web pixels.

Maybe just StandardEvents could be type aliases of base event?

Reply:
This change has been implemented to try to go more in the direction of making it easier for the developers so they just have one base type for the events. I will try to create a "BaseCustomEvent" without duplicating "BaseEvent".